src/worker.js

```diff
@@ -136,6 +138,7 @@ onmessage = initMessage => {
       importScripts,
       postMessage,
       createImageBitmap,
+      FileReader
```

Comment:
Are we gonna have `FileReader` here?

Reply:
Is this a question for me? I thought I was following your instruction by placing it here

Reply:
Yeah, I'm trying to understand how `FileReader` can be used here since it's not imported above.

Reply:
However, having it here does seem correct.